src/components/Popups/ReplyField/DefaultItemRow.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import DatalistItem from 'box-ui-elements/es/components/datalist-item';
+import { UserMini, GroupMini } from '../../../@types';
+import './DefaultItemRow.scss';
+
+export type Props = {
+    id?: string;
+    item?: UserMini | GroupMini;
+    name?: string;
+};
+
+const DefaultItemRow = ({ item, ...rest }: Props): JSX.Element | null => {
+    if (!item || !item.name) {
+        return null;
+    }
+
+    return (
+        <DatalistItem {...rest}>
+            <div className="ba-DefaultItemRow-name">{item.name}</div>
+            {'email' in item && <div className="ba-DefaultItemRow-email">{item.email}</div>}
```

Comment:
is `'email' in item` preferred over `item.email`?

Reply:
> is `'email' in item` preferred over `item.email`?

TS may throw error if use `item.email` directly, cause `item` may be `MiniGroup` type which doesn't have `email` property at all. `'email' in item` is the TS way to check.

Reply:
`item?.email`? lol